Cargo.toml

```diff
@@ -13,21 +13,18 @@ doctest = false
 [features]
 unstable = [
     "string_cache/unstable",
-    "tendril/unstable",
     "rc/unstable",
```

Comment:
It seems "rc/unstable" can be replaced with stabilized `Weak` element. Perhaps in another PR?

mbrubeck did it for me in Ygg01/xml5ever/pull/12